### DISCUSSION TITLE
Editable Columns Plugin -> not installable with backpack 6

### DISCUSSION THREAD

**Discussion by de-raaf-media** [Laravel-Backpack/community-forum#563] [Q&A (Help)]
Hi,

I was upgrading my clients website to backpack 6. But it seems editable columns is not yet compatible? Think I might need a version bump?

Regards
Mark

**Comment by pxpm** [comment]
Hey @de-raaf-media . 

You should use version `^3.0` in your `composer.json` . 

You can see here the package changelog: https://backpackforlaravel.com/products/editable-columns/CHANGELOG.md
and here all the backpack v6 package version compatibility: https://backpackforlaravel.com/docs/6.x/upgrade-guide#step-2

Let me know if you still have issues.

Cheers

**Reply by de-raaf-media** [reply]
Hi @pxpm seems I've missed that completely. Sorry about that!
